8Knot/assets/color.css

```diff
@@ -62,28 +93,121 @@ blockquote p {
   text-align: center;
 }
 
-.glace_headers {
-  color: #000000 !important;
-  text-shadow: none !important;
-  font-weight: "normal" !important;
+/* Headers with higher specificity */
+.glace-container .glace_headers,
+div.glace_headers {
+  color: var(--color-black);
+  text-shadow: none;
+  font-weight: normal;
 }
 
-/* changes the color of an alert */
-.alert-warning {
-    background-color: #B5B682 !important;
+/* ==== BOOTSTRAP COMPONENT OVERRIDES ==== */
+/* Alert warning with higher specificity */
+.alert.alert-warning {
+  background-color: var(--color-secondary);
 }
 
-/* updates background of the dmc-multiselect in visualizations */
+/* ==== DASH MANTINE COMPONENTS ==== */
+/* DMC multiselect custom styling */
 .dmc-multiselect-custom {
-  background-color: #272b30;
+  background-color: var(--color-card-bg);
 }
 
-/* updates background color of switch*/
-.botlist-filter-switch:checked {
-  background-color: #6b8976;
+/* ==== FORM COMPONENTS ==== */
+/* Switch styling with higher specificity */
+.filter-container .botlist-filter-switch:checked,
+input.botlist-filter-switch:checked {
+  background-color: var(--color-primary);
 }
 
-/* updates popover size to allow more text per line */
+/* Popover styling */
 .popover {
   max-width: 500px;
 }
+
+/* ==== CARD COMPONENTS ==== */
+/* Card title with higher specificity */
+.card h3.card-title,
+div.card h3.card-title {
```

Comment:
These electors aren't that different from each other. I feel like `.card .card-title` may work better since its not limited to specific elements. Does doing that break the CSS by not being specific enough to override base dash styles?